src/Makefile.am

```diff
@@ -235,23 +235,23 @@ souffle_sources = \
         interpreter/InterpreterRelation.h                  \
         parser.cc                                          \
         parser.hh                                          \
-        ram/RamCondition.h                                 \
-        ram/RamExpression.h                                \
-        ram/RamNode.h                                      \
-        ram/RamOperation.h                                 \
-        ram/RamProgram.h                                   \
-        ram/RamRelation.h                                  \
-        ram/RamStatement.h                                 \
-        ram/RamTranslationUnit.h                           \
-        ram/RamUtils.h                                     \
-        ram/RamVisitor.h                                   \
-        ram/analysis/RamAnalysis.h                         \
-        ram/analysis/RamComplexityAnalysis.cpp             \
-        ram/analysis/RamComplexityAnalysis.h               \
-        ram/analysis/RamIndexAnalysis.cpp                  \
-        ram/analysis/RamIndexAnalysis.h                    \
-        ram/analysis/RamLevelAnalysis.cpp                  \
-        ram/analysis/RamLevelAnalysis.h                    \
+        ram/Condition.h                                 \
+        ram/Expression.h                                \
+        ram/Node.h                                      \
+        ram/Operation.h                                 \
+        ram/Program.h                                   \
+        ram/Relation.h                                  \
+        ram/Statement.h                                 \
+        ram/TranslationUnit.h                           \
+        ram/Utils.h                                     \
+        ram/Visitor.h                                   \
+        ram/analysis/Analysis.h                         \
+        ram/analysis/ComplexityAnalysis.cpp             \
```

Comment:
`Analysis` could be removed from the filenames of the analyses in `ram/analysis`

Reply:
Let's do this in another step.

Reply:
There are many files with not quite suitable filenames.